src/pages/StatusPage/StatusPage.tsx

```diff
@@ -14,7 +14,7 @@ export const StatusPage = () => {
   const { transactionId } = useParams();
 
   const { data: transactionData } = useFetch({
-    url: transactionId ? `/transaction/${transactionId}` : "",
+    url: transactionId ? `transaction/${transactionId}/` : "",
```

Comment:
the right  URI is `/transactions/`
```suggestion
    url: transactionId ? `transactions/${transactionId}/` : "",
```